src/RADOS/RADOSImpl.cc

```diff
@@ -0,0 +1,117 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2004-2012 Sage Weil <sage@newdream.net>
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ *
+ */
+
+
+#include <boost/system/system_error.hpp>
+
+#include "common/common_init.h"
+
+#include "global/global_init.h"
+
+#include "RADOSImpl.h"
+
+namespace RADOS {
+namespace _ {
```

Comment:
I get the convention and even like it, but please consider "detail", "internal", etc., as I think these will be both more familiar to folks (especially if they're not familiar with ML-family languages) and easier to search for/spot. Just my $0.25!

Reply:
Fixed in other PR...